src/common/utils/deploy.utils.ts

```diff
@@ -58,13 +59,36 @@ export default class DeployUtils {
     return flags;
   }
 
-  static getComponentSecrets(individual_secrets: string[], secrets_file?: string): Dictionary<Dictionary<string | number | null>> {
-    const component_secrets = DeployUtils.readSecretsFile(secrets_file);
+  static getAllSecretFiles(raw_argv: any): string[] {
```

Comment:
This function would probably just become.
```
const secret_files = flags.values.concat(flags['secret-files']);
```

Reply:
Ended up removing since we can pass multiple secret files in